app/metrics.yaml

```diff
@@ -4121,8 +4123,10 @@ collections:
     extra_keys:
       tabs_open:
         description: The number of tabs open in the current session
+        type: quantity
```

Comment:
Making these `quantity` will this cause the data to look different comparing to the original?

Reply:
The original was indeed recorded as `string`.